frontend/src/common/components/Card/index.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Card, CardBodyStyled } from './styled';
+import { CardProps } from './types';
+import { HiOutlineArrowRight } from 'react-icons/hi';
+
+// Example components
+const Cards: React.FC<CardProps> = ({ ...props }) => {
```

Comment:
Spread props อ่านง่ายกว่านะครับ
```suggestion
const Cards: React.FC<CardProps> = ({ primary, title, description, date }) => {
```

Reply:
หรือถ้าจะไม่ spread ก็เขียนแบบนี้ไปเลยครับ
```suggestion
const Cards: React.FC<CardProps> = (props) => {
```